tools/lsmcli/cmdline.py

```diff
@@ -806,8 +806,8 @@ def _get_item(l, the_id, friendly_name='item', raise_error=True):
     ['ar', 'access-group-remove'],
     ['ad', 'access-group-delete'],
     ['vri', 'volume-raid-info'],
-    ['vils', 'volume-ident-led-set'],
-    ['vilc', 'volume-ident-led-clear'],
+    ['vilon', 'volume-ident-led-on'],
```

Comment:
Ended up needing to make a judgement call on this one, since both are technically 'vilo'. Lemme know if you have other preferences.